tests/test_hebrew_date.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
```

Comment:
Can you please add test cases to Adar A and Adar B?

Reply:
i add :)